primitives/api/proc-macro/src/decl_runtime_apis.rs

```diff
@@ -539,8 +539,6 @@ impl<'a> Fold for ToClientSideDecl<'a> {
 			input.supertraits.push(parse_quote!( #crate_::Core<#block_ident> ));
 		}
 
-		// The client side trait is only required when compiling with the feature `std` or `test`.
-		input.attrs.push(parse_quote!( #[cfg(any(feature = "std", test))] ));
```

Comment:
Removal of test is intentional?

Reply:
Yes, it was basically done to ensure that someone may not declares the `std` feature for their crate. However, this was not really working as otherwise this pr would not have been required.